src/three-domain-secure/utils.jsx

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+/** @jsx node */
+/* eslint max-lines: 0 */
+
+import { node, dom } from "@krakenjs/jsx-pragmatic/src";
+import { create, type ZoidComponent } from "@krakenjs/zoid/src";
+import { inlineMemoize, noop } from "@krakenjs/belter/src";
+import { getCSPNonce, getClientID, getSDKMeta } from "@paypal/sdk-client/src";
+
+import { payPayDomainRegEx } from "../lib";
+import { Overlay } from "../ui/overlay";
+
+import type { TDSProps } from "./types";
+
+export type TDSComponent = ZoidComponent<TDSProps>;
+
+export function getFastlaneThreeDS(): TDSComponent {
+  return inlineMemoize(getFastlaneThreeDS, () => {
+    const component = create({
+      tag: "fastlane-threeds",
+      url: ({ props }) => props.payerActionUrl,
+
+      attributes: {
+        iframe: {
+          scrolling: "no",
+        },
+      },
+
+      containerTemplate: ({
+        context,
+        focus,
+        close,
+        frame,
+        prerenderFrame,
+        doc,
+        event,
+        props,
+      }) => {
+        return (
+          <Overlay
+            context={context}
+            close={close}
+            focus={focus}
+            event={event}
+            frame={frame}
+            prerenderFrame={prerenderFrame}
+            content={props.content}
+            nonce={props.nonce}
+          />
+        ).render(dom({ doc }));
+      },
+      domain: payPayDomainRegEx,
```

Comment:
We should remove this and use the getPayPalDomainRegex() method that sdk-client provides.